jwst/regtest/test_nircam_tsimg.py

```diff
@@ -71,18 +71,14 @@ def test_nircam_setpointing_tsimg(rtdata, engdb, fitsdiff_default_kwargs):
     NIRCam TSO imaging file.
     """
     # Get SIAF PRD database file
-    siaf_path = rtdata.get_data("common/prd.db")
-    rtdata.get_data("nircam/tsimg/jw00312006001_02102_00001-seg001_nrcb1_uncal.fits")
+    rtdata.get_data("nircam/tsimg/jw06780001001_02103_00001-seg002_nrcblong_uncal.fits")
```

Comment:
This file appears to have been taken very recently - is it public?

Reply:
<img width="695" alt="Screenshot 2024-10-14 at 2 58 18 PM" src="https://github.com/user-attachments/assets/ae6084cd-2f17-426d-a65a-ccb6748775a6">
Yep!

One more code commit just made to clear up some unused mock db machinery, and I'll get a fragment and regtests running.